src/CreateBoardForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component, FormEvent } from 'react'
+import { remove, shuffle } from 'lodash'
+import { Checkbox } from './Checkbox'
+
+export class CreateBoardForm extends Component<{}, { chosenCharacters: string[], showBoard: boolean, board: string[] }> {
+  baseCharacters: string[] = [
+    "Arya Stark",
+    "Sansa Stark",
+    "Bran Stark",
+    "Jon Snow",
+    "Theon Greyjoy",
+    "Yara Greyjoy",
+    "Euron Greyjoy",
+    "Cersei Lannister",
+    "Jaime Lannister",
+    "Tyrion Lannister",
+    "Sam Tarly",
+    "Tormund Giantsbane",
+    "Brienne of Tarth",
+    "Gendry",
+    "Podrick",
+    "Danaerys Targaryen",
+    "Jorah Mormont",
+    "Davos Seaworth",
+    "Lyanna Mormont",
+    "The Night King"
+  ]
+
+  optionalCharacters: string[] = [
+    "Lord Varys",
+    "Missandei",
+    "Grey Worm",
+    "The Hound",
+    "Beric Dondarrion",
+    "Bronn",
+    "Melisandre",
+    "Edd Tollett",
+    "Gilly",
+    "Little Sam",
+    "Drogon",
+    "Rhaegal"
+  ]
+
+  state = {
+    chosenCharacters: [] as string[],
+    showBoard: false,
+    board: [] as string[]
+  }
+
+  handleCheckboxChange = (event: FormEvent): void => {
+    let { chosenCharacters } = this.state
+    const target = event.target as HTMLInputElement
+
+    if (target.checked) {
+      chosenCharacters.push(target.name)
+    } else {
+      remove(chosenCharacters, (character: string): boolean => character === target.name)
+    }
+
+    this.setState({ chosenCharacters })
+  }
+
+  shuffleBoard = (): void => {
+    const board: string[] = shuffle(this.baseCharacters.concat(this.state.chosenCharacters))
+    this.setState({ showBoard: true, board })
+  }
+
+  render() {
+    const buttonDisabled: boolean = this.state.chosenCharacters.length + this.baseCharacters.length !== 25
```

Comment:
magic number, move it to a constant

Reply:
Ugh I know but come on it's bingo, those rules aren't changing any time soon